example/demo/models.py

```diff
@@ -120,7 +120,8 @@ def upload_path_handler(instance, filename):
         ("4.3.0.15", "4.3.0.15"),
         ("4.4.0.58", "4.4.0.58"),
         ("4.4.0.64", "4.4.0.64"),
-        ("4.4.0.68", "4.4.0.68")
+        ("4.4.0.68", "4.4.0.68"),
+        ("4.4.0.68", "4.4.0.72")
```

Comment:
hi, you have a typo, both values should be 72

Reply:
> hi, you have a typo, both values should be 72

Damnit, I can't even get a oneliner patch right. I really should be replaced with a shell script.

Fixed in the fixup, pls squash.